diff --git a/2024/LobbyInfoData.h b/2025/LobbyInfoData.h
old mode 100755
new mode 100644
index ebbfa7e..5176263
--- a/2024/LobbyInfoData.h
+++ b/2025/LobbyInfoData.h
@@ -4,7 +4,7 @@ struct LobbyInfoData
  uint8 m_teamId; // Team id - see appendix (255 if no team currently selected)
  uint8 m_nationality; // Nationality of the driver
  uint8 m_platform; // 1 = Steam, 3 = PlayStation, 4 = Xbox, 6 = Origin, 255 = unknown
-    char      m_name[48];	  // Name of participant in UTF-8 format – null terminated
+ char m_name[32]; // Name of participant in UTF-8 format – null terminated
  // Will be truncated with ... (U+2026) if too long
  uint8 m_carNumber; // Car number of the player
  uint8 m_yourTelemetry; // The player's UDP setting, 0 = restricted, 1 = public
