diff --git a/2023/PacketHeader.h b/2024/PacketHeader.h
old mode 100755
new mode 100644
index 1364554..3dc325c
--- a/2023/PacketHeader.h
+++ b/2024/PacketHeader.h
@@ -1,7 +1,7 @@
 struct PacketHeader
 {
-    uint16    m_packetFormat;            // 2023
-    uint8     m_gameYear;                // Game year - last two digits e.g. 23
+    uint16    m_packetFormat;            // 2024
+    uint8     m_gameYear;                // Game year - last two digits e.g. 24
     uint8     m_gameMajorVersion;        // Game major version - "X.00"
     uint8     m_gameMinorVersion;        // Game minor version - "1.XX"
     uint8     m_packetVersion;           // Version of this packet type, all start from 1
